ru/thesaurus.md

```diff
@@ -0,0 +1,14 @@
+| Термин     | Определение | Примечание |
```

Comment:
I suggest to use sections rather than a table, a section per notion.

For instance,

```
# Термины и определения

## Тезаурус

Специальный толковый словарь, охватывающий понятия, определения и термины, формирующие язык и профессиональную культуру специалистов, работающих в сфере информационных технологий.

## Система

Комплекс взаимодействующих компонентов, обособленный от окружающей среды и взаимодействующий с ней, как единое целое.
```

It is more convenient both for editing and for referencing it from other docs. The main use is to be able to create a kind of local version of a thesaurus for a specific project, or just referece a term in docs of projects.

```